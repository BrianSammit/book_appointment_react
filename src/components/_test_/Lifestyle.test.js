/**
 * @jest-environment jsdom
 */

import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Lifestyle from '../Lifestyle';

describe('Creates a component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <Lifestyle />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Renders a component', () => {
    render(
      <Router>
        <Lifestyle />
      </Router>,
    );
    const element = screen.getByText('STAKE OR DIE');
    expect(element).toBeInTheDocument();
  });
});
