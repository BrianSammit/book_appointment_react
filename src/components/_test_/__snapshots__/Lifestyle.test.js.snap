// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a component renders correctly 1`] = `
<div
  className="lifestyle-container"
>
  <div
    className="lifestyle-square"
  >
    <div
      className="lifestyle-div-1"
    />
    <div
      className="lifestyle-div-2 div-color"
    >
      <h5>
        HERITAGE
      </h5>
      <i
        className="fas fa-angle-right"
      />
    </div>
    <div
      className="lifestyle-div-3"
    />
    <div
      className="lifestyle-div-4 div-color"
    >
      <h5>
        THE LOVER
      </h5>
      <i
        className="fas fa-angle-right"
      />
    </div>
    <div
      className="lifestyle-div-5"
    />
    <div
      className="lifestyle-div-6 div-color"
    >
      <h5>
        PRESTIGE
      </h5>
      <i
        className="fas fa-angle-right"
      />
    </div>
    <div
      className="lifestyle-div-7"
    />
    <div
      className="lifestyle-div-8 div-color"
    >
      <h5>
        STAKE OR DIE
      </h5>
      <i
        className="fas fa-angle-right"
      />
    </div>
    <div
      className="lifestyle-div-9"
    />
  </div>
</div>
`;
